node-export-server/server.js

```diff
@@ -256,12 +259,14 @@ function doExport(req, res, hash, input) {
         if (isFile) {
             command += ' convert';
             if (bbox) command += ' -D ' + bbox_param + '=' + bbox;
+            if (req.query.overrideTags) command += ' -D tags.override=' + overrideTags;
```

Comment:
Looks like the param is `translation.override`

Reply:
And the overrideTags json definitely need to be escaped.  I'm getting `Error running convert: convert takes two parameters.` with this command logged:

>hoot convert -D convert.bounding.box=-72.49812,18.54086,-72.49374,18.54270 -D translation.override={"attribution":"","security:resource_owner":"","security:classification":"","security:dissemination:control:ic":"","security:dissemination:control:non_ic":"","security:releasability":"","source:":"","source:non_spatial_source:type":"","source:copywrite":""} "osmapidb://vagrant:vagrant!@192.168.33.12:5432/openstreetmap" /home/vagrant/hoot/node-export-server/export_63edf3a0-f496-11e7-b1ae-2f5428121067.osm